src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputOperatorWrap.java

```diff
@@ -104,39 +104,37 @@ class Inner {
             for (Map.Entry<String, String> entry :
                 map.entrySet())
             {
-                /*warn*/int i = flag == true ?
+                int i = flag == true ? // violation ''?' should be on a new line.'
                         1 : 2;
             }
 
-            /*warn*/ if (init !=
+             if (init != // violation ''!=' should be on a new line.'
                     9)
             {
 
             }
 
-            /*warn*/ while (init ==
+             while (init == // violation ''==' should be on a new line.'
                     10)
             {
 
             }
 
-            /*warn*/ if (init >
+             if (init > // violation ''>' should be on a new line.'
                     10)
             {
 
             } else {}
 
-            /*warn*/ while (init < 10 ||
+             while (init < 10 || // violation '.* should be on a new line.'
                     !flag) {
 
             }
         }
     }
 
-    Inner anon = new Inner
-            (){
-        void testGenerics1
-        ()
+    Inner anon = new Inner(){
+        void testGenerics1()
```

Comment:
Why did you reformat it? Could you use the original formatting?

Reply:
It was a violation of `MethodParamPad` module, this scenario has been tested in its own input file, I have kept the violations only related to `OperatorWrap` in this input file. So I had to reformat it. Pls refer this: https://github.com/checkstyle/checkstyle/pull/15190#pullrequestreview-2157980882